apis/apis.json

```diff
@@ -468,7 +468,7 @@
     "id": "Google.Cloud.OsLogin.V1",
     "productName": "Google Cloud OS Login API",
     "productUrl": "https://cloud.google.com/compute/docs/instances/managing-instance-access",
-    "version": "1.0.0-beta01",
+    "version": "1.0.0-beta02",
```

Comment:
Because we're updating this and it has a "project" dependency on Google.Cloud.OsLogin.Common, we need to update the Common one above too. (The tagreleases code would have warned about this, don't worry :) Will explain in detail in a VC

Reply:
Done.